texastribune/talk
# Remove unwanted plugins

## What does this PR do?
Removes the respect and permalink plugins per audience request.

## How do I test this PR?
This branch is merged into `test`, so you can view a comment embed on https://tt-auth-test.texastribune.org and confirm it has neither the permalink button nor the respect button.

This branch is itself a branch off `auth0`, so the latter should be merged first.
